internal/global/state.go

```diff
@@ -50,14 +50,12 @@ func SetTracerProvider(tp trace.TracerProvider) {
 	delegateTraceOnce.Do(func() {
 		current := TracerProvider()
 		if current == tp {
-			// Setting the provider to the prior default is nonsense, panic.
-			// Panic is acceptable because we are likely still early in the
-			// process lifetime.
-			panic("invalid TracerProvider, the global instance cannot be reinstalled")
+			// Setting the provider to the prior default results in a noop. Return
+			// early.
+			return
```

Comment:
Sure. Should I reuse the globalLogger defined in `internal_logging.go`?
Metrics has no such logger. Should I add one?

Reply:
> Sure. Should I reuse the globalLogger defined in `internal_logging.go`?

Yeah, that seems to make sense to me.

> Metrics has no such logger. Should I add one?

Unifying on a single logger is going to be the goal. The metrics global package is able to import this internal package. It might take exposing the logger in the internal package. Recreating the internal logging logic in the metrics package should not be done.

Reply:
Also, I mistakenly said to log this with an info level, but the linked issue says we had decided to log this as an error. I would stick with logging this as an error (as it is one).

Reply:
Done.